app/views/template_syncs/template_attrs.json.rabl

```diff
@@ -0,0 +1,11 @@
+object @template
+
+attributes :id, :snippet, :locked
+
+node(:kind) do |template|
+  template.template_kind.name if template.respond_to?(:template_kind) && template.template_kind
+end
+
+node(:class_name) do |template|
+  template.class.name.underscore.split('_').map { |part| part.capitalize }.join(' ')
```

Comment:
Can we change this, "Ptable" isn't as friendly to users as it could be?

Reply:
Good point, changed.